scaffolder-templates/github-pages-site/template.yaml

```diff
@@ -89,7 +89,7 @@ spec:
       action: http:backstage:request
       input:
         method: 'POST'
-        path: /api/proxy/github-api/repos/${{ (parameters.repoUrl | parseRepoUrl)["owner"] }}/${{ (parameters.repoUrl | parseRepoUrl)["repo"] }}/pages
+        path: /api/proxy/github/api/repos/${{ (parameters.repoUrl | parseRepoUrl)["owner"] }}/${{ (parameters.repoUrl | parseRepoUrl)["repo"] }}/pages
```

Comment:
It'd be great if there was some notion of "computed parameters" (not sure if that's the right term) so the owner value could be reference more succinctly (and also reused).

Reply:
That's a good idea. Maybe we can mention that to @kissmikijr